src/components/TopPicks/TopPicks.tsx

```diff
@@ -68,8 +77,41 @@ const linkStyles = css`
   }
 `;
 
+const columWrapperStyles = css`
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+`;
+const colLeft = css`
+  padding-right: 10px;
+`;
+const colRight = css`
+  padding-left: 10px;
+`;
+
+const picksWrapper = css`
+  max-width: 620px;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const twoColComments = css`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  ${until.tablet} {
+    display: none;
+  }
+`;
+const oneColComments = css`
```

Comment:
The names `oneColComments` and `twoColComments` are very similar to `leftColComments` and `rightColComments` making is seem like they're the same but actually they're very different.

Maybe oneColumnStyles and twoColumStyles?